### PR TITLE
save load system

### PR DESCRIPTION
**Pull request by M-125**
ur "art" will be saved when clicking the quit button
all u have to do is to trigger load() method of a layoutsaver node (probably from main menu) to load /create canvas to put sprites on

### CODE REVIEW

**Comment by M-125** [comment]
saving happens on pressing quit button and doesnt need another layoutsaver node
(static func)

File: scripts/Menu.gd [hunk 1]
```diff
@@ -62,12 +66,12 @@ func _on_file_dialog_file_selected(path):
 		if err != OK:
 			printerr("cannot load image.")
 			return
-		Save.filepath = path
 		openingfor.texture = ImageTexture.create_from_image(image)
 
 
 func _on_drag_requested(object: ScreenObject):
-	if drag_target == object:
+	print("req")
```
Comment (quellus) [inline scripts/Menu.gd]:
```suggestion

```

File: scenes/MicRecord.tscn [hunk 2]
```diff
@@ -169,7 +169,7 @@ unique_name_in_owner = true
 title = "Open a File"
 initial_position = 1
 size = Vector2i(682, 589)
-ok_button_text = "Open"
+ok_button_text = "Megnyitás"
```
Comment (quellus) [inline scenes/MicRecord.tscn]:
```suggestion
ok_button_text = "Open"
```

File: scripts/save.gd
Comment (quellus) [inline scripts/save.gd:1]: Rename `save.gd` to `layout_saver.gd` so it has the same name as the class.

**Review by quellus — CHANGES_REQUESTED** [review]
I want to get this working with the main scene `MicRecord.tscn` before we merge.
It seems like we either need to add the save script somewhere in the scene or make it an autoload since the `load()` function can't be made static.

**Comment by quellus** [comment]
Closes #26

**Comment by M-125** [comment]
What did i forget to do that this still cant be merged

**Pull request closed without merging**